colossalai/amp/__init__.py

```diff
@@ -13,14 +13,15 @@
 __all__ = ['convert_to_amp', 'convert_to_naive_amp', 'convert_to_apex_amp', 'convert_to_torch_amp', 'AMP_TYPE']
 
 
-def convert_to_amp(model: nn.Module, optimizer: Optimizer, criterion: _Loss, mode: AMP_TYPE, amp_config: Config = None):
+def convert_to_amp(model: nn.Module, optimizer: Optimizer, criterion: _Loss, mode: AMP_TYPE, use_bf16: bool = False, amp_config: Config = None):
```

Comment:
passing a bool paramter use_bf16 is not a good way. could you passing a precision type? you can use .to(torch.bfloat16) or .to(torch.float16) to control your target precision type.